components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -3912,7 +3912,7 @@ void RewardsServiceImpl::SetTransferFee(
     new_wallet = wallet->Clone();
   }
 
-  auto* fees = wallet->FindListKey("transfer_fees");
+  auto* fees = wallet->FindDictKey("transfer_fees");
```

Comment:
fees will be a `nullptr` if `transfer_fees` does not exist in the dictionary. Not an issue for this PR but we should pick these up in future PRs.

Reply:
we check fees object on line 3917

Reply:
Damn I must get some more sleep...